### PR TITLE
Migrate to Icarus

### PR DESCRIPTION
**Pull request by ArmaanT**
This PR migrates labs-api-server to icarus.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Install project dependencies
+RUN pipenv install --system
+
+# Copy project files
+COPY . /app/
+
+
+# Run uWSGI
+CMD ["/usr/local/bin/uwsgi", "--ini", "/app/setup.cfg"]
```
Comment (pawalt) [inline Dockerfile]: does this docker image serve anything static, and if so, do we need mime types?

Comment (ArmaanT) [reply Dockerfile]: I don't think there's anything static being served

File: k8s/values.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        path: "/app/ios_key.p8"
+  - name: redis
+    image: redis
+    tag: 6.0rc
+    port: 6379
+cronjobs:
+  - name: laundry
+    schedule: "*/15 * * * *"
+    secret: labs-api-server
+    image: pennlabs/labs-api-server:latest
```
Comment (pawalt) [inline k8s/values.yaml]: remove latest from these and bump icarus version to .14

Comment (ArmaanT) [reply k8s/values.yaml]: Fixed

**Pull request merged**